### ISSUE TITLE
Training schedule question

### ISSUE THREAD

**Issue by alexwq100** [xingyizhou/CenterNet#465]
hi,@xingyizhou,
I have 2 questions related to the training:
1. I notice that you use adam to train the model on coco dataset, and the total training epoch is about 140 ,right? I try to training it by using a 8 gpus of p100, and the training time is about 3 days. I wonder did you try sgd to train the model in coco? and why the total epoch is 140 in contrast to 12 epoch for maskrcnn ?
2.  in your configuration ,the input image size is 512*512,and the batch size is 128 for coco dataset training. while , for maskrcnn, the batch size is 16,and the input image size is 800*1333. what is the reason you choose smaller image size and large batchsize instead of larger image size and small batch size

**Comment by xingyizhou** [comment]
Hi,
  Thanks for the good questions.
  1. You are right. The 140-epoch schedule follows the MSRA human pose baseline (https://github.com/microsoft/human-pose-estimation.pytorch). I haven't explored that before the submission. After the submission, I found training 70 epochs with lr dropped at the 60th epoch works equally well. I do have tried SGD on the 70-epoch schedule , it can achieve similar  AP with 5x learning rate. 
  2. Most of my current configs are from CornerNet. I haven't tried 800x1333 resolution. The comparison to MaskRCNN is something I am not clear and planed to explore soon.

**Comment by alexwq100** [comment]
thank you for the quick reply! Do you think the large image size will be beneficial to  the performance?

**Comment by xingyizhou** [comment]
I tried 640x640, it trains slow and gives similar or worse results than 512x512 so I stop trying. On reason preventing using 800x1333 is the aspect ratio. MaskRCNN uses a dynamic input size as the batchsize is 2, but it is not possible for batchsize 32x.

**Comment by alexwq100** [comment]
"it is not possible for batchsize 32x." do you mean the image size must be dividable by 32?

**Comment by xingyizhou** [comment]
I mean FasterRCNN use "min edge 800, the other edge < 1333". In practice the actual input shape is usually less than 800x1333 or 1333x800. However in our case the input shape should be the max of all images in a batch, in this case it will be almost always 800x1333. You also need to decide to use 800x1333 or 1333x800.

**Comment by alexwq100** [comment]
if  800x1333 is used , do you have the result on it ?

**Comment by xingyizhou** [comment]
I haven't tried.
